.github/workflows/build_and_deploy.yml

```
@@ -87,3 +87,9 @@ jobs:
         with:
           command: app set iam-cache-server
           options: -p image.tag=${{needs.unique_id.outputs.unique_id}}
+      - name: ArgoCD overvrite values.yaml
```

@hejkerooo Could you add a bit to this suggested comment to explain the why?
```suggestion 
      # For OCI based helm charts, tag needs to be overwritten.. {why??}
      - name: ArgoCD overwrite values.yaml
```